activesupport/lib/active_support/core_ext/yaml.rb

```diff
@@ -0,0 +1,17 @@
+require "yaml"
+
+module YAML
```

Comment:
We should not add a monkey patch to implement this feature. We don't want to implement a new version of the YML specification. Could you move it back to the previous implementation and only add support to the database configuration?